Console/Command/QueueShell.php

```diff
@@ -11,7 +11,9 @@
 
 App::uses('Folder', 'Utility');
 App::uses('File', 'Utility');
-App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
+if (!class_exists('HipChat\HipChat')) {
```

Comment:
Just wrapping this in a conditional feels a bit like a hack. Do we have any idea why it would be included previously?